concrete/src/Database/DatabaseManagerORM.php

```diff
@@ -1,18 +1,16 @@
 <?php
+
 namespace Concrete\Core\Database;
 
 use Concrete\Core\Application\Application;
-use Concrete\Core\Database\Connection\Connection;
-use Concrete\Core\Package\Package;
 use Doctrine\ORM\EntityManager;
-use Database;
 
 class DatabaseManagerORM
 {
     /**
      * The application instance.
      *
-     * @var Application
+     * @var \Concrete\Core\Application\Application
```

Comment:
I'm not sure I understand the purpose of changing Application to the FQN, when Application is being imported already.

Reply:
@aembler There are mainly two reasons why I also prefer FQN in phpdocs:

1. the `c5:c5:ide-symbols` CLI command copies the phpdocs of the original classes when it generates the `__IDE_SYMBOLS__.php` file, so if in the phpdocs we use aliases defined by `use` statements in the original file, they won't work in the `__IDE_SYMBOLS__.php` file (so, the IDEs can't understand the method return types)
2. when humans read a phpdoc, it's much easier to see the FQN instead of having to look at the top of the file.  
For example, if you read a method like this:  
   ```php
   /**
    * @return UrlInterface
    */
   public function getURL()
   {
   }
   ```  
   you have to look at the top of the file to see if the return value is `Concrete\Core\Url\UrlInterface` or `League\Url\UrlInterface`. Things get even worse when there's no `use` statements (because we are already in the same namespace as the used class), or when the `use` statements define aliases.

Reply:
Because it's not a default PhpStorm setting; here is where it can be configured:

![afbeelding](https://user-images.githubusercontent.com/1431100/49440508-2919af80-f7c4-11e8-90eb-acac4426ad4e.png)